lib/themes/dosomething/paraneue_dosomething/js/utilities/ApiClient.js

```diff
@@ -18,7 +18,8 @@ class ApiClient {
     this.host = host;
     this.port = port === '' ? null : port;
     this.version = `/api/${version}/`;
-    this.url = `//${this.host}${this.port ? ':' + this.port : ''}${this.version}`;
+    this.domain = `//${this.host}${this.port ? ':' + this.port : ''}`;
```

Comment:
Yeah thats a trick Diego taught me, lets you skip defining http or https (Which caused problems while developing onboarding locally i think)